docs/content/tutorial/backups.md

```diff
@@ -123,6 +128,59 @@ kubectl apply -k kustomize/s3
 
 Watch your cluster: you will see that your backups and archives are now being stored in S3!
 
+### Using an AWS-integrated identity provider and role
+
+If you deploy PostgresClusters to AWS Elastic Kubernetes Service, you can take advantage of their 
+IAM role integration. When you attach a certain annotation to your PostgresCluster spec, AWS will 
+automatically mount an AWS token and other needed environment variables. These environment 
+variables will then be used by pgBackRest to assume the identity of a role that has permissions 
+to upload to an S3 repository.
+
+This method requires [additional setup in AWS IAM](https://docs.aws.amazon.com/eks/latest/userguide/iam-roles-for-service-accounts.html).
+Follow the procedure there, but skip the third step:
+
+1\. Create an OIDC provider for your EKS cluster
+2\. Create an IAM policy for bucket access and an IAM role with a trust relationship with the 
+OIDC provider in step 1.
+
+The third step is to associate that IAM role with a ServiceAccount, but there's no need to 
+do that manually, as PGO does that for you. First, make a note of the IAM role's `ARN`.
+
+You can then make the following changes:
+
+1\. Add the `s3` section to the spec in `kustomize/s3/postgres.yaml` as discussed in the 
```

Comment:
:thinking: Maybe a link here to the [example](https://github.com/CrunchyData/postgres-operator-examples/tree/main/kustomize/s3)?